packages/python-yara-dex/PKGBUILD

```diff
@@ -0,0 +1,38 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=python-yara-dex
+_pkgname=yara-python
```

Comment:
Is that not possible to use [this template](https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/PKGBUILD-python-lib-setuptools) instead of what you did manually? It seems it is available on pypi.

Reply:
I'm having issues figuring out the proper URL for the pypi source, see : 
![image](https://user-images.githubusercontent.com/14914796/206875967-f6cf7fc1-aa68-4c9b-9a0c-dd0554fae144.png)
This doesn't seem to exist : https://files.pythonhosted.org/packages/source/y/yara-python-dex/yara-python-dex-1.0.3.tar.gz
Even if it exists here : https://pypi.org/project/yara-python-dex/

If you can help me see, what is the one thing I'm doing wrong?
The source is defined as:
```bash
https://files.pythonhosted.org/packages/source/${_pkgname::1}/$_pkgname/$_pkgname-$pkgver.tar.gz
```
I've also tried : 
```sh
https://files.pythonhosted.org/packages/py3/${_name::1}/$_pkgname/${_pkgname//-/_}-$pkgver-py3-none-any.whl
```

Reply:
So what should we do ? Do I keep the git sources ? What should I do  regarding the source URL?

Reply:
As it's not on pypi, the git tags from the git repo could be used instead: https://github.com/MobSF/yara-python-dex/archive/refs/tags/1.0.3.tar.gz

It's just that the auto-update won't work.

Reply:
I'm running into an issue, I'm not sure why : 
```
Traceback (most recent call last):
  File "/home/username/project/opensource/blackarch/packages/python-yara-dex/src/yara-python-dex-1.0.3/setup.py", line 310, in <module>
    with open('yara-python/README.rst', 'r', 'utf-8') as f:
  File "/usr/lib/python3.10/codecs.py", line 905, in open
    file = builtins.open(filename, mode, buffering)
FileNotFoundError: [Errno 2] No such file or directory: 'yara-python/README.rst'
==> ERROR: A failure occurred in build().
```
Here is the tarball contents, it seems the `setup.py` code tries to read a file that will never be at the right place?
![image](https://user-images.githubusercontent.com/14914796/208267021-c37f5690-5ed9-49ec-9f36-7da247f19148.png)
I'm pushing along the changes, I've made, so you can take a look, but I'd like your input on what you would recommend, or maybe I'm doing something wrong (using `git+https://` worked fine, but I believe that using the release tag, it should work all the same).

Oh, I got it, it's because it's using a git submodule : 
![image](https://user-images.githubusercontent.com/14914796/208267107-3ea10517-be03-4863-b377-8bdb78a44b60.png) and the tarball does not contain the git submodule's files, so should I get the latest release tag of the git submodule as well ?

Reply:
Ah yeah, git sub-modules are a pain. As there is no source archive on pypi and the tarball on github release has not `.git` that is required for git sub-modules, then the best way would be to do a git style PKGBUILD like https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/PKGBUILD-generic-git

Reply:
I'm sorry it's a pain, you didn't pick the easiest for a first contribution but I'm glad continue working on it. 😉

Reply:
I got around to fix it, just recently, if you could review the changes, thanks !